websites/A/AniWorld/presence.ts

```diff
@@ -49,73 +53,75 @@ presence.on(
 );
 
 presence.on("UpdateData", async () => {
+	if (!strings) strings = await getStrings();
+
 	const page = document.location.pathname,
 		presenceData: PresenceData = {
-			largeImageKey: "anicloud",
+			largeImageKey: "aniworld-logo",
 			startTimestamp: Math.floor(Date.now() / 1000)
 		};
 	if (page === "/") presenceData.details = "Betrachtet die Startseite";
 	else if (page.startsWith("/anime/")) {
-		presenceData.details = `${
-			(document.querySelector("h1") as HTMLElement).textContent
-		}`;
-		presenceData.state = `${(
-			document.querySelector("title") as HTMLElement
-		).textContent
-			.split("Staffel")[0]
-			.replace("Filme von", " ")
-			.split(" | AniCloud.io - Animes gratis online ansehen")} - ${
-			(document.querySelector("h2") as HTMLElement).textContent
-		}`;
-		[presenceData.startTimestamp, presenceData.endTimestamp] =
-			presence.getTimestamps(currentTime, timeEnd);
-		presenceData.buttons = [
-			{
-				label: "Watch Anime",
-				url: document.location.href
-			}
-		];
+		// Check if we are on the episode selection page
+		if (page.split("/").length === 4) {
+			presenceData.details =
+				document.querySelector<HTMLHeadingElement>("h1").textContent;
+			presenceData.state = "Betrachtet die Episodenliste";
+		} else {
+			presenceData.details =
+				document.querySelector<HTMLHeadingElement>("h1").textContent;
+			presenceData.state = `${document
+				.querySelector<HTMLTitleElement>("title")
+				.textContent.split("Staffel")[0]
+				.replace("Filme von", " ")
+				.split(" | AniWorld.to - Animes gratis online ansehen")} - ${
+				document.querySelector<HTMLHeadingElement>("h2").textContent
+			}`;
+			[presenceData.startTimestamp, presenceData.endTimestamp] =
+				presence.getTimestamps(currentTime, timeEnd);
+			presenceData.buttons = [
+				{
+					label: "Watch Anime",
+					url: document.location.href
+				}
+			];
 
-		video = document.querySelector("video");
-		if (video) {
-			played = video.currentTime !== 0;
-			timeEnd = video.duration;
-			({ currentTime } = video);
-			({ paused } = video);
-		}
-		if (played) {
-			if (!paused) {
-				[, presenceData.endTimestamp] = presence.getTimestamps(
-					currentTime,
-					timeEnd
-				);
+			video = document.querySelector<HTMLVideoElement>("video");
+			if (video) {
+				played = video.currentTime !== 0;
+				({ currentTime, duration: timeEnd, paused } = video);
+			}
+			if (played) {
+				if (!paused) {
+					[, presenceData.endTimestamp] = presence.getTimestamps(
+						currentTime,
+						timeEnd
+					);
+				}
+				presenceData.smallImageKey = paused ? "pause" : "play";
+				presenceData.smallImageText = paused ? strings.pause : strings.play;
```

Comment:
This PR is meant to be a rebrand not a refactor from what I understand. If you want to completely overhaul the presence then you should probably change the title of the PR. Otherwise I would recommend undoing these

Reply:
I changed the PR title. Not sure how to describe it, since english is not my main language, hope this is okay.

Reply:
so did the entire website change or did only the URL change?

Reply:
only the url as far i'm aware, but during testing I noticed a few things that were broken (based on urls that existed previously that are no longer existing). Those we're fixed a few commits ago